File: app/src/main/java/org/ionchain/wallet/bean/MoreWalletBean.java
package org.ionchain.wallet.bean;

import java.io.Serializable;
import java.util.List;

/**
 * USER: binny
 * DATE: 2018/9/13
 * 描述:
 */
public class MoreWalletBean implements Serializable{

    private static final long serialVersionUID = -9157094035840945437L;
    private List<WalletBean> mWallets;

    public List<WalletBean> getWallets() {
        return mWallets;
    }

    public void setWallets(List<WalletBean> wallets) {
        mWallets = wallets;
    }
}

File: app/src/main/java/org/ionchain/wallet/bean/MoreWalletBean.kt
package org.ionchain.wallet.bean

import org.sdk.wallet.bean.WalletBeanNew
import java.io.Serializable

/**
 * USER: binny  596928539@qq.com
 * DATE: 2018/9/13
 * 描述:
 */
class MoreWalletBean : Serializable {
    var wallets: List<WalletBeanNew>? = null

    companion object {

        private const val serialVersionUID = -9157094035840945437L
    }
}
